reference/docs-conceptual/install/Installing-PowerShell-Core-on-Windows.md

```diff
@@ -95,9 +95,9 @@ PowerShell binary ZIP archives are provided to enable advanced deployment scenar
 ZIP archive doesn't check the prerequisites like the MSI packages do. For remoting over WSMan to
 work properly, ensure that you've met the [prerequisites](#prerequisites).
 
-## Deploying on Windows IoT
+## Deploying on Windows 10 IoT Enterprise
```

Comment:
When you changed this H2 you broke a link in powershell-core-on-arm.md. Can you update the bookmark link in powershell-core-on-arm.md?

Reply:
Updated.